pkg/query/args.go

```diff
@@ -214,13 +213,6 @@ func (a *Args) Prepare(writers ...io.Writer) (*Statement, error) {
 		return s, err
 	}
 
-	// check external calls
-	if a.External {
```

Comment:
Ah, nice. I wasn't sure when going over this in #111 whether it was still needed at OSAG.